src/app/app/onboarding/_components/PlanSelection.tsx

```
@@ -0,0 +1,206 @@
+  "3 messages per month",
+  "3 applications per month",
+  "Basic profile",
+];
+
+export function PlanSelection({ pricing, onSelectFree }: PlanSelectionProps) {
+  const [billingInterval, setBillingInterval] = useState<"monthly" | "annual">("monthly");
+  const [error, setError] = useState<string | null>(null);
+
+  const createCheckout = api.subscription.createCheckoutSession.useMutation({
+    onMutate: () => setError(null),
+    onError: (err) => {
```

ESLint

[error] Expected object keys to be in natural insensitive ascending order. 'onError' should be before 'onMutate'.

(sort-keys-fix/sort-keys-fix)